File: components/types/api/music-response.ts
import { Alphabets } from "./alphabets.enum";

export interface MusicResponse {
  artist: string;
  artistAlphabetCategory: Alphabets;
  id: string;
  isFavorite: boolean;
  length: number;
  path: string;
  title: string;
  album: string;
}

File: Edge.MyMusic/src/Edge.MyMusic/Controllers/Models/MusicResponse.cs
using Edge.MyMusic.Shared;

namespace Edge.MyMusic.Controllers.Models;

public sealed class MusicResponse
{
#nullable disable
    public string Album { get; init; }
    public string Artist { get; init; }
    public AlphabetType ArtistAlphabetCategory { get; init; }
    public string Id { get; init; }
    public bool IsFavorite { get; init; }
    public int Length { get; init; }
    public string Path { get; init; }
    public string Title { get; init; }
#nullable enable
}